lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -229,19 +249,39 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
             (process: ProcessDefinitionCloud) => process.name === selectedProcessDefinitionName || process.key === selectedProcessDefinitionName
         );
 
-        this.startProcessCloudService.getStartEventFormStaticValuesMapping(this.appName, processDefinitionCurrent.id).subscribe(
-            (staticMappings) => {
-                this.staticMappings = staticMappings;
-                this.resolvedValues = this.staticMappings.concat(this.values || []);
-                this.processDefinitionCurrent = processDefinitionCurrent;
-                this.isFormCloudLoading = false;
-            },
-            () => {
-                this.resolvedValues = this.values;
-                this.processDefinitionCurrent = processDefinitionCurrent;
-                this.isFormCloudLoading = false;
+        forkJoin([
+            this.startProcessCloudService
+                .getStartEventFormStaticValuesMapping(this.appName, processDefinitionCurrent.id)
+                .pipe(catchError(() => of([] as TaskVariableCloud[]))),
+            this.startProcessCloudService
+                .getStartEventConstants(this.appName, processDefinitionCurrent.id)
+                .pipe(catchError(() => of([] as TaskVariableCloud[])))
+        ]).subscribe(([staticMappings, constants]) => {
+            this.staticMappings = staticMappings;
+            this.resolvedValues = this.staticMappings.concat(this.values || []);
+            this.processDefinitionCurrent = processDefinitionCurrent;
+            this.isFormCloudLoading = false;
+
+            const displayStart = constants?.find((constant) => constant.name === 'startEnabled');
+            const startLabel = constants?.find((constant) => constant.name === 'startLabel');
+
+            const displayCancel = constants?.find((constant) => constant.name === 'cancelEnabled');
+            const cancelLabel = constants?.find((constant) => constant.name === 'cancelLabel');
+
+            if (displayStart) {
+                this.displayStartProcessButton = displayStart?.value === 'true';
             }
```

Comment:
Are the ifs here and below necessary if we're using optional chaining?

Reply:
yes they are, because for example, if the display start i not set, then `displayStartProcessButton` should be true, and we can't do 
```typescript
this.displayStartProcessButton = displayStart?.value === 'true' || true; 
```
because that's always true